client/src/components/LogStatus.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+// SASS
+import styles from './../sass/components/LogStatus.module.scss';
+
+const LogStatus = (props) => {
+
+  console.log(props)
+  return (
+    <div className={styles.logStatus}>
+      <div className={styles.imageContainer} >
+        <img src={props.imageURL} className={styles.image} alt="Avatar" /> 
+      </div>
+      <div className={styles.textContainer}>
+        <h3> Your neighbour {props.firstName} - Self isolated today! </h3>
```

Comment:
Change to `<h3> Your neighbour {props.firstName} self-isolated today! </h3>` as self-isolated is a compound adjective